Update permit_user_env comments in config ref (#18813)
gravitational/teleport: docs/pages/reference/config.mdx

      command: ['/bin/uname', '-p']
      period: 1h0m0s

>     # Enables reading ~/.tsh/environment on the server before creating a session.
>     # By default it's set to false can be set true here or through the
>     # command-line flag.
>     permit_user_env: false

Reviewer: While we're here, this doesn't read quite right.

```suggestion
    # Enables reading ~/.tsh/environment on the server before creating a session.
    # Disabled by default. Can we enabled here or via the `--permit-user-env` flag.
    # command-line flag.
    permit_user_env: false
```

      command: ['/bin/uname', '-p']
      period: 1h0m0s

    # Enables reading ~/.tsh/environment on the server before creating a session.

>     # Disabled by default. Can we enabled here or via the `--permit-user-env` flag.

Reviewer 2: ```suggestion
    # Disabled by default. Can be enabled here or via the `--permit-user-env` flag.
```
